pt/reference/linuxtools.rst

```diff
@@ -1,28 +1,27 @@
-Phalcon Developer Tools on Linux
+Ferramentas Phalcon Developer em Linux
 ================================
 
-These steps will guide you through the process of installing Phalcon Developer Tools for Linux.
+Esses passos irá guiá-lo através do processo de instalação de Ferramentas Phalcon Developer para Linux.
 
-Prerequisites
+Pré-requisitos
 -------------
-The Phalcon PHP extension is required to run Phalcon Tools. If you haven't installed it yet, please see the :doc:`Installation <install>`
-section for instructions.
+A extensão Phalcon PHP é necessário para executar Phalcon Tools. Se você não tiver instalado ainda, por favor consulte o :doc:`Intalação <install>` para obter instruções.
 
 Download
 --------
-You can download a cross platform package containing the developer tools from the Download_ section. Also you can clone it from Github_.
+Você pode baixar um pacote de plataforma que contém as ferramentas de desenvolvedor da seção de Download. Também você pode cloná-lo a partir de Github_.
```

Comment:
The word "Download" needs to have an underscore after it ("Download_") so it can be a link.